<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StackRoom</class>
 <widget class="QWidget" name="StackRoom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel,
QComboBox,
QPushButton 
{
	border: none;
	color: #1296db;
	font: 14px &quot;微软雅黑&quot;;
}

QTableView
{
	border: none;
	background-color: rgba(255, 255, 255, 0.7);
	color: #000000;
	font: 14px &quot;微软雅黑&quot;;
}

QTableView:item:selected
{
	color: #1296db;
	background: qlineargradient(spread: pad, x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #def0fe, stop: 1 #c0def6);
}

QTableView:item:hover
{
	background: qlineargradient(spread: pad, x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #f2f9ff, stop: 1 #daefff);
}

QTableView:item
{
	padding: 5px;
	margin: 0px;
}

QScrollBar:vertical
{
	padding-top: 15px;
	padding-bottom: 15px;
	background-color: rgba(0, 0, 0, 0);
}

QScrollBar:add-line:vertical
{
	height: 15px;
	width: 15px;
	subcontrol-position: bottom;
	subcontrol-origin: margin;
	border-image: url(:/Images/down.png);
}

QScrollBar:add-line:vertical:pressed
{
	height: 15px;
	width: 15px;
	subcontrol-position: bottom;
	subcontrol-origin: margin;
	border-image: url(:/Images/down_pre.png);
}

QScrollBar:sub-line:vertical
{
	height: 15px;
	width: 15px;
	subcontrol-position: top;
	subcontrol-origin: margin;
	border-image: url(:/Images/up.png);
}

QScrollBar:sub-line:vertical:pressed
{
	height: 15px;
	width: 15px;
	subcontrol-position: top;
	subcontrol-origin: margin;
	border-image: url(:/Images/up_pre.png);
}

QScrollBar:handle:vertical
{
	background: #1296db;
	width: 15px;
}

QScrollBar:handle:vertical:hover
{
	width: 15px;
	background: #00aaff;
}

QLineEdit
{
	border: none;
	color: #000000;
	font: 75 14px &quot;微软雅黑&quot;;
	border-radius: 5px;
	selection-background-color: #def0fe;
	selection-color: #1296db;
}

QComboBox
{
	border-radius: 3px;
}

QComboBox::drop-down
{	
	border-left-width: 1px;
	border-left-style: solid;	
	border-bottom-right-radius: 3px;
	border-left-color: #1296db;
}

QComboBox:down-arrow
{
	image: url(:/Images/down.png);
}

QPushButton:pressed
{
	color: #ffffff;
}

QMessageBox
{
	border-image: url(:/Images/dialog.jpg);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="MenuBar" name="widget_4" native="true"/>
   </item>
   <item>
    <widget class="QLabel" name="bookInfoLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
	background-color: rgba(255, 255, 255, 0.7);
    border-radius: 4px;  
	color: #1296db;
	font: 14px &quot;微软雅黑&quot;;
}</string>
     </property>
     <property name="text">
      <string>图书信息</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="bookInfoView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #000000;</string>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideLeft</enum>
     </property>
     <property name="showGrid">
      <bool>true</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="queryTypeLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel
{
	background-color: rgba(255, 255, 255, 0.7);
    border-radius: 4px;  
	color: #1296db;
	font: 14px &quot;微软雅黑&quot;;
}</string>
        </property>
        <property name="text">
         <string>查询类型</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="queryTypeComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <item>
         <property name="text">
          <string>编号</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>书名</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>出版社</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>作者</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="queryEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>320</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="placeholderText">
         <string>请输入查询信息</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="borrowButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>30</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton 
{
	background-color: rgba(18, 150, 219, 0.7);
    border-radius: 4px;  
	color: #ffffff;
	font: 14px &quot;微软雅黑&quot;;
}  

QPushButton:pressed
{
	background-color: #00aaff;
}
</string>
        </property>
        <property name="text">
         <string>借阅书籍</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MenuBar</class>
   <extends>QWidget</extends>
   <header>menubar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
